### DISCUSSION TITLE
发音评估时报错 账户有余额

### DISCUSSION THREAD

**Discussion by ClM12345** [ZuodaoTech/everyone-can-use-english#835] [Q&A]
![IMG_20240719_065832](https://github.com/user-attachments/assets/6ac4cf6f-da29-4906-a6b0-3b8f6a2009e6)

**Comment by an-lee** [comment]
这个是网络问题。重试一下看？

**Reply by ClM12345** [reply]
更新到 新版 换了api 好了
